### PR TITLE
Add aria-label to Social Media in footer for Accessibility

### PR DESCRIPTION
**Pull request by discodavey**

### CODE REVIEW

File: source/_patterns/01-molecules/components/social-links.mustache [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="social-links">
+<div class="social-links" role="contentinfo" aria-label="Social media links for eLife Sciences">
```
Comment (davidcmoulton) [inline source/_patterns/01-molecules/components/social-links.mustache]: I wonder if the`contentinfo` is redundant here, as the footer is already the contentinfo in this context: https://w3c.github.io/aria-practices/examples/landmarks/contentinfo.html#tabpanel2 (The `social-links` pattern is only ever in the site footer.)

Comment (discodavey) [reply source/_patterns/01-molecules/components/social-links.mustache]: I see what you mean and agree but as usual there are developers out there who think you should still add it in.

I will remove the role and leave the label if you're happy with that?

Comment (thewilkybarkid) [reply source/_patterns/01-molecules/components/social-links.mustache]: Could use a comment to clarify why it’s not there?

File: source/_patterns/01-molecules/components/social-links.mustache [hunk 2]
```diff
@@ -1,3 +1,4 @@
+<!-- No aria role contentinfo needed in Social Links as this is always placed within the footer -->
```
Comment (thewilkybarkid) [inline source/_patterns/01-molecules/components/social-links.mustache]: Use Mustache comments (`{{! ... }}`), as we don't actually want this appearing in the real HTML.

**Pull request merged**